### PR TITLE
fix: tooltip optimize && i18n

### LINKED ISSUE #4137: [Bug] Optimize tooltip content of NFTs binding rule
### How to reproduce?

Screen Shot 2021-08-27 at 16 16 55

### Mask Version:

- [x] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by BillySh1**
closes #4137

### CODE REVIEW

File: packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx [hunk 1]
```diff
@@ -50,6 +58,27 @@ export function NFTPage(props: NFTPageProps) {
...
     const { loading: loadingWalletGun, value: walletAddressGun } = useUserOwnerAddress(identity.identifier.userId)
 
+    const rulesTipMap = [
+        t('plugin_profile_binding_rule1'),
+        t('plugin_profile_binding_rule2'),
+        t('plugin_profile_binding_rule3'),
+        t('plugin_profile_binding_rule4', { suffix: `".eth"` }),
+    ]
+
+    const tooltipRender = (
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx:68]: Sorry forgotten to say that MUI has the Tooltip component.

Learn more: https://github.com/DimensionDev/Maskbook/pull/4814

File: packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx [hunk 2]
```diff
@@ -50,6 +58,27 @@ export function NFTPage(props: NFTPageProps) {
...
+        <div>
+            <div style={{ fontSize: '0.8rem' }}>{t('plugin_profile_binding_rules_title')}</div>
+            {rulesTipMap.map((item) => {
+                return (
+                    <div>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx:73]: Lost `key` here.

Comment (BillySh1) [reply packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx:73]: done

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Review by Jack-Works — APPROVED** [review]

**Comment by guanbinrui** [comment]
@jk234ert Hide it on mobile devices?

**Review by Jack-Works — DISMISSED** [review]

**Pull request merged**